tasks/apt.yml

```diff
@@ -135,8 +135,7 @@
 
 - name: Install fping
   when: openwisp2_monitoring is true
-  apt:
-    - fping
+  apt: name=fping
```

Comment:
could you share info about changes from old syntax to the new one?
We have quite a lot of this so if it still works we could avoid changing it now and change it all at once later on.